ta4j-core/src/main/java/org/ta4j/core/Position.java

```diff
@@ -215,6 +215,26 @@ public boolean isNew() {
     public String toString() {
         return "Entry: " + entry + " exit: " + exit;
     }
+    
+    /**
+     * @return true if position is closed and {@link #getProfit()} > 0
+     */
+    public boolean isWinning() {
```

Comment:
Nice! 
I did almost same. Except I used Optional<Boolean>;  in case the position is still open, Optional.empty is returned.

Reply:
If position is open, false is returned (because Position#getProfit() always returns `0` for open positions). It's the **realized** profit or loss. Optional is not needed here.